docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+
+## Usage
+
+Prebuild can be used by running:
+
+<Terminal cmd={[
+'$ npx expo prebuild',
+]} cmdCopy="npx expo prebuild" />
+
+This creates the `ios/` and `android/` folders for running your React code. If you modify the generated directories manually then you risk losing your changes the next time you run `npx expo prebuild --clean`. Instead, you should create [Expo Config Plugins][config-plugins] which modify the native directories safely during prebuild.
+
+We highly recommend using Prebuild for the reasons listed in the [pitch](#pitch) section, but the system is [fully optional](#optionality) and you can stop using it at any time.
+
+## Platforms
+
+Prebuild currently supports iOS and Android, web support is not required as the analogous native app is the web browser. You can build for individual platforms by using the `--platform` flag:
+
+<Terminal cmd={[
+'$ npx expo prebuild --platform ios',
+]} cmdCopy="npx expo prebuild --platform ios"/>
+
+## Dependencies
+
+Prebuild requires specific packages and versions are used in order to ensure the template works. The packages are inferred from the `dependencies` field in the [template](#templates) `package.json` file.
+
+You can skip changing NPM package versions with the `--skip-dependency-update` flag:
+
+<Terminal cmd={[
+'$ npx expo prebuild --skip-dependency-update react-native,react',
+]} cmdCopy="npx expo npx expo prebuild --skip-dependency-update react-native,react"/>
+
+## Package Manager
+
+When the [dependencies](#dependencies) are changed, prebuild will reinstall packages using the inferred package manager that is currently used in the project. Specific package managers can be used by providing one of: `--npm`, `--yarn`, `--pnpm`.
+
+All installation can be skipped by passing the `--no-install` command, this is useful for testing generation quickly.
+
+## Clean
+
+The `--clean` flag will clear any existing native directories before generating, this is the safest way to use the prebuild command, but it's also the slowest to recompile as you clear some generated project caches.
+
+When you re-run `npx expo prebuild`, it layers changes on top of your existing files to make rebuilding the native app faster. This is purely for convenience and starts to breakdown when your project utilizes a lot of "dangerous modifiers" for performing changes like adding regex changes to application code.
+
+Due to the pivotal nature of the flag, you'll be warned to have a clean git status when the `--clean` flag is used. This prompt is optional and will be skipped when encountered in CI.
+
+You can disable the check by enabling the environment variable `EXPO_NO_GIT_STATUS=1` if you'd like.
+
+The purpose of the prompt is to encourage managed workflow users to add the `/ios` and `/android` folders to the project's `.gitignore`, ensuring that the project is always managed. This can however make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
```

Comment:
```suggestion
The purpose of the prompt is to encourage managed workflow users to add the `/ios` and `/android` folders to the project's `.gitignore`, ensuring that the project is always managed. However, this can make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
```